### DISCUSSION TITLE
BOM Products

### DISCUSSION THREAD

**Discussion by chrisBradley-dev** [unopim/unopim#67] [Ideas]
It would be great if you could create a BOM/assembly within a product of components needed. even if it was a list of input products and quantities.

**Comment by navneetkumar-pim-webkul** [comment]
@chris-chameleonbrands 
Thank you for your suggestion! The idea of creating a BOM (Bill of Materials) or assembly within a product is excellent and aligns well with enhancing product management capabilities. Adding a list of input products with their respective quantities could be valuable for scenarios involving complex product assemblies or kits.

We’ll evaluate the feasibility of implementing this feature in future updates, considering its potential benefits and alignment with our roadmap. Meanwhile, if you have specific use cases or requirements, please share them—they can help us design the feature more effectively.

**Reply by chrisBradley-dev** [reply]
@navneetbhardwaj 

Thanks. 

I think for a mass-user perspective, something like the associations tab on the righthand side of products, but including a quantity value would probably be ideal. from a compatibility standpoint, the ability to link into shopfront shopify native bundles would be ideal. 

from an API perspective, the ability to query the product and get an array of products within the bom with ideally
- id
- sku
- quantity
- maybe title
